test/benchmark_vochain_test.go

```diff
@@ -0,0 +1,26 @@
+package test
```

Comment:
@mvdan can you check the basics of this benchmark and maybe propose improvements? The idea is to have a full benchmark to test performance of state+app+indexer+api

Reply:
I think the logic looks okay. Two suggestions come to mind:

* What kind of election is this - encrypted or not? We should probably benchmark the one that is most common. If both are common, then I would suggest that the benchmark do _both_ side by side. We could similarly think about other kinds of votes, like overwrites.

* You use a sequential benchmark via `b.N`, which is fine, but not very realistic - in reality, people won't vote in a big election in a big line :) For that reason, I would use a parallel benchmark via https://pkg.go.dev/testing#B.RunParallel. You can still use `b.N` as the total number of iterations to set up the census and anything else you need, but the looping should use https://pkg.go.dev/testing#PB.Next instead.

Reply:
Also, should the benchmark advance votes while iterating? Because again, in reality, you won't have an entire election get all of its votes within a single block. Plus, the indexer does a significant amount of its work in the `Commit` method, so you should advance at least one block while the benchmark timer is on.